### PR TITLE
added my folder

### PR DESCRIPTION
**Pull request by lundinjosefin**

### CODE REVIEW

File: van-gogh-gallery/josefin/index.html [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+</head>
+
+<body>
+    <header>
+        <nav>
+            <div class="nav-container">
+                <li><a href="/home" class="nav-link">Home</a></li>
+                <li><a href="#" class="nav-link">Tickets & Visit</a></li>
+                <li><a href="#" class="nav-link">Art & Stories</a></li>
+                <li><a href="pages/about.html" class="nav-link">About</a></li>
```
Comment (interstates21) [inline van-gogh-gallery/josefin/index.html:19]: 👍🏻👍🏻👍🏻

File: van-gogh-gallery/josefin/index.html [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+                <li><a href="#" class="nav-link">Webstore</a></li>
+                <li><a href="#" class="nav-link">NL|EN</a></li>
+                <li><a href="#" class="nav-link">Search</a></li>
+            </div>
+
+        </nav>
+        <div id="van-gogh-logo">
+            <img src="/images/logo.svg" alt="logo" width="110">
```
Comment (interstates21) [inline van-gogh-gallery/josefin/index.html]:
```suggestion
            <img src="./images/logo.svg" alt="logo" width="110">
```

The path has a typo

File: van-gogh-gallery/josefin/index.html [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+        <div class="last-card">
+            <h2 class="title-regular">
+                Discover more
+            </h2>
+            <ul>
+
+
+                <a href="#"> > Search the collection</a><br>
+                <a href="#"> > Vincent´s Artist Friends</a><br>
+                <a href="https://www.vangoghmuseum.nl/en/japanese-prints"> > Vincent's Key Influences</a>
```
Comment (interstates21) [inline van-gogh-gallery/josefin/index.html]:
```suggestion
                <a href="https://www.vangoghmuseum.nl/en/japanese-prints" target="_blank"> > Vincent's Key Influences</a>
```

File: van-gogh-gallery/josefin/pages/about.css [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+}
+
+a{
+    text-decoration: none;
+    color: black;
+}
+
+
+body {
+    font-family: 'Montserrat', sans-serif;
```
Comment (interstates21) [inline van-gogh-gallery/josefin/pages/about.css]: No need to specify it for the second time, cause you've already specified it in `* {. .. }`

File: van-gogh-gallery/josefin/pages/about.css [hunk 5]
```diff
@@ -0,0 +1,191 @@
...
+
+.nav-container{
+    position: absolute;
+   
+    margin-left: 0px;
+    margin-bottom: 10px;
+    cursor: pointer;
+    transition: all 0.2s;
+    font-size: 20px;
+font-weight: bolder;
```
Comment (interstates21) [inline van-gogh-gallery/josefin/pages/about.css:18]: This won't work cause your import link contains only font-weight: 200 `@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@200&display=swap');`

File: van-gogh-gallery/josefin/pages/about.css [hunk 6]
```diff
@@ -0,0 +1,191 @@
...
+}
+
+li{
+    display: inline-block;
+
+}
+li a{
+  text-decoration: none; 
+  color: black;
+  font-family: 'Montserrat', sans-serif;;
```
Comment (interstates21) [inline van-gogh-gallery/josefin/pages/about.css]: Typo 
```suggestion
  font-family: 'Montserrat', sans-serif;
```

File: van-gogh-gallery/josefin/pages/about.css [hunk 7]
```diff
@@ -0,0 +1,191 @@
...
+  color: black;
+  font-family: 'Montserrat', sans-serif;; 
+  font-weight: lighter ;
+  font-size: 22px;
+  padding: 0 20px;
+  padding-left: 40px;
+  font-weight: 900;
+}
+
+.nav-link::after {
```
Comment (interstates21) [inline van-gogh-gallery/josefin/pages/about.css]: I think we don't use this after element anywhere 🤔

File: van-gogh-gallery/josefin/pages/about.css [hunk 8]
```diff
@@ -0,0 +1,191 @@
...
+    content: "";
+    display: block;
+    width: 0;
+    height:  2px;
+    background: black;
+    transition: width .3s 
+}
+
+.nav-link :hover::after {
```
Comment (interstates21) [inline van-gogh-gallery/josefin/pages/about.css]: Same here

File: van-gogh-gallery/josefin/pages/about.css [hunk 9]
```diff
@@ -0,0 +1,191 @@
...
+
+a {
+    background-color: none;
+    text-decoration: none;
+    color: inherit;
+
+}
+li{
+    list-style: none;
+transition: all 0,4s;
```
Comment (interstates21) [inline van-gogh-gallery/josefin/pages/about.css]:
```suggestion
transition: all 0.4s;
```

File: van-gogh-gallery/josefin/pages/about.html [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="about.css">
+    <title>about</title>
+</head>
+
+<body>
+    <header>
+        <nav>
+            <div class="nav-container">
+                <li><a href="/van-gogh-gallery/josefin/index.html" class="nav-link">Home</a></li>
```
Comment (interstates21) [inline van-gogh-gallery/josefin/pages/about.html]:
```suggestion
                <li><a href="../index.html" class="nav-link">Home</a></li>
```
I think it's more readable

File: van-gogh-gallery/josefin/pages/about.html [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+                <li><a href="#" class="nav-link">Tickets & Visit</a></li>
+                <li><a href="#" class="nav-link">Art & Stories</a></li>
+                <li><a href="pages/about.html" class="nav-link">About</a></li>
```
Comment (interstates21) [inline van-gogh-gallery/josefin/pages/about.html]: We're alredy on about 
```suggestion
                <li><a href="#" class="nav-link">About</a></li>
```

File: van-gogh-gallery/josefin/style.css [hunk 12]
```diff
@@ -0,0 +1,263 @@
...
+}
+
+li{
+    display: inline-block;
+
+}
+li a{
+  text-decoration: none; 
+  color: black;
+  font-family: 'Montserrat', sans-serif;;
```
Comment (interstates21) [inline van-gogh-gallery/josefin/style.css]:
```suggestion
  font-family: 'Montserrat', sans-serif;
```

**Review by interstates21 — COMMENTED** [review]
Good job! That's great that you've added a second page

File: van-gogh-gallery/josefin/pages/about.css (hunk 7 above)
Comment (lundinjosefin) [reply van-gogh-gallery/josefin/pages/about.css]: oh i think i tried somethin out and then forgot to delete all of it :o

File: van-gogh-gallery/josefin/pages/about.css (hunk 5 above)
Comment (lundinjosefin) [reply van-gogh-gallery/josefin/pages/about.css:18]: updated

**Review by interstates21 — APPROVED** [review]
Perfect

**Pull request merged**